src/HypothesisClient/Credentials/Credentials.php

```diff
@@ -2,56 +2,61 @@
 
 namespace eLife\HypothesisClient\Credentials;
 
-use Serializable;
+use Firebase\JWT\JWT;
 
-class Credentials implements Serializable
+class Credentials
 {
-    private $clientId;
-    private $secret;
+    private $userManagement;
+    private $jwtSigning;
     private $authority;
+    private $group;
 
-    public function __construct(string $clientId, string $secret, string $authority)
+    public function __construct(UserManagementCredential $userManagement, JWTSigningCredential $jwtSigning, string $authority, string $group)
```

Comment:
This class seems to be doing multiple things: the two different sets of credentials don't need to be tied together.